issue-009/solution.js

```diff
@@ -1,4 +1,14 @@
 // implement your solution here
 function sumEven(a) {
-
+	//create new array, from given array, with only even numbers
+	var arrEven = [];
+	a.forEach(function(element) {
```

Comment:
With this, you will end up doing 2 loops -- one for `.forEach` and another for `.reduce`. You are to use `.reduce` only. How will you approach it?